doc/nrf/known_issues.rst

```diff
@@ -504,6 +504,12 @@ Subsystems
 Bluetooth LE
 ============
 
+.. rst-class:: v1-6-0 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0 v1-1-0 v1-0-0
+
+NCSDK-9820: The :ref:`peripheral_lbs` - while holding one of the other buttons, the notification for button release is the same as for button press.
```

Comment:
This is a bit unclear now when referring one of the other buttons. Which button release gives the same notification as the button press. I guess it is the button 1. We could perhaps use the description from the Jira ticket:
"If **button 1** pushed and released while holding one of the other buttons, the notification for release is the same as for press"

Reply:
Right. Changed